tests/0330-oasis_C_Verify.phpt

```diff
@@ -0,0 +1,172 @@
+--TEST--
+OASIS Verify Basic test - C_VerifyInit(), C_Verify()
+--DESCRIPTION--
+Based on the same design than C_Sign() tests, let's first
+C_Sign() a data then let's C_Verify().
+--SKIPIF--
+<?php
+
+require_once 'require-module-load.skipif.inc';
+
+if (getenv('PHP11_MODULE') === false) {
+  echo 'skip';
+}
+
+$info = $module->getInfo();
+
+if (trim($info["manufacturerID"]) == 'SoftHSM')
+  echo 'skip: CKO_PRIVATE_KEY/CKK_RSA, missing with SoftHSM';
```

Comment:
SoftHSM most certainly supports RSA keys and operations. Why this check ?

Reply:
Because it did not work the way it is implemented by this test which applies to some HSM that already have the Key objects without using C_GenerateKeyPair() neither C_CreateObject().